src/components/Dashboard/Dashboard.jsx

```diff
@@ -51,24 +49,31 @@ export class Dashboard extends React.Component {
     }
     render() {
         const startGameButton = <button id="start-game" onClick={this.startGame}>Start game</button>;
-        const nextTurnButton = (this.props.gameState.myTurn) ? <button id="end-turn" onClick={this.endTurn}>End my turn</button> : "";
-
+        const nextTurnButton = (this.props.myTurn) ? <button id="end-turn" onClick={this.endTurn}>End my turn</button> : "";
```

Comment:
This can be applied to the two lines below:
```suggestion
        const nextTurnButton = this.props.myTurn && <button id="end-turn" onClick={this.endTurn}>End my turn</button>;
```